lib/workers/repository/update/pr/pr-cache.ts

```diff
@@ -0,0 +1,43 @@
+import { logger } from '../../../../logger';
+import { getCache } from '../../../../util/cache/repository';
+import type { PrCache } from '../../../../util/cache/repository/types';
+
+export function getPrCache(branchName: string): PrCache | null {
+  logger.debug(`getPrCache()`);
+  const cache = getCache();
+  const branch = cache.branches?.find(
+    (branch) => branchName === branch.branchName
+  );
+
+  if (branch?.prCache) {
+    return branch.prCache;
+  }
+
+  return null;
```

Comment:
```suggestion
  if (is.nullOrUndefined(branch?.prCache)) {
    return null;
  }
  return branch.prCache;
```

Reply:
Using this as TS cannot infer that branch is not undefined when using `is`
```ts
  if (!branch?.prCache) {
    return branch.prCache;
  }

  return null;
```